src/Sulu/Bundle/TagBundle/Resources/config/routing_api.yaml

```diff
@@ -0,0 +1,55 @@
+sulu_tag.get_tag:
+    path: /tags/{id}.{_format}
+    controller: sulu_tag.tag_controller::getAction
+    methods: GET
+    format: json
```

Comment:
I don't know what this `format` key is supposed to do. If you want to specify what values it can have it should be something like
```
sulu_tag.get_tag:
    requirements: { _format: json|csv }
```

Reply:
format does the same as options: _format so its obsolete if already defined in options